src/components/MovieCard/MovieCard.tsx

```diff
@@ -0,0 +1,45 @@
+import classnames from 'classnames';
+import { useState } from 'react';
+
+import styles from './MovieCard.module.scss';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEllipsisVertical } from '@fortawesome/free-solid-svg-icons';
+
+export interface MovieCardProps {
+  title: string;
+  poster: string;
+  realiseDate: number;
+  genre: string;
+}
+
+export const MovieCard = (props: MovieCardProps) => {
+  const { genre, poster, title, realiseDate } = props;
+  const [isMovieCardWithHover, setIsMovieCardWithHover] = useState(false);
+
+  const iconClass = classnames(
+    `${styles.movieCardCircle}`,
+    !isMovieCardWithHover ? `${styles.movieCardCircleHidden}` : '',
```

Comment:
use { [styles]: condition } for classnames

Reply:
done, thanks